src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsassessforearlyreleaseapi/event/PrisonOffenderEventListener.kt

```diff
@@ -8,15 +8,8 @@ import org.springframework.stereotype.Service
 import uk.gov.justice.digital.hmpps.hmppsassessforearlyreleaseapi.service.OffenderService
 import uk.gov.justice.digital.hmpps.hmppsassessforearlyreleaseapi.service.TransferPrisonService
 
-fun interface EventProcessingCompleteHandler {
-  fun complete()
-}
-
-val NO_OP = EventProcessingCompleteHandler { }
-
 @Service
 class PrisonOffenderEventListener(
-  private val done: EventProcessingCompleteHandler = NO_OP,
   private val offenderService: OffenderService,
```

Comment:
Should avoid code like this, as we are altering the production code to test the production code! So have removed it and tested the actually production code to see if it has completed.